web/views/root_view.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Designator.RootView do
```

Comment:
What's this view doing that the controller isn't already?

Reply:
I copied this from the StatusController - it's really just a view like rails has. I could have used the `render_json_response` controller helper as well. 
https://github.com/zooniverse/designator/blob/8cca637fd058c1c43616b73fb84030a501f244cf/web/controllers/status_controller.ex#L4